crates/nu-std/std/formats.nu

```diff
@@ -0,0 +1,18 @@
+# formats.nu
+#
+# This file contains functions for formatting data in various ways.
+#
+# Usage:
+#   use std format *
+#   use std format <function name>
+#
+# These functions help `open` the files with unsupported extensions such as ndjson.
+#
+
+export def "from ndjson" []: string -> any {
```

Comment:
maybe we could add a little help to both these commands, so that someone can do
```nushell
use std formats
help formats
```
```nushell
use std formats "from ndjson"
help from ndjson
```

Reply:
thanks for the help doc :pray: